torch/nn/parallel/distributed.py

```diff
@@ -1236,6 +1236,13 @@ def _ddp_init_helper(
         # passing a handle to torch.nn.SyncBatchNorm layer
         self._passing_sync_batchnorm_handle(self.module)
 
+    def __getattr__(self, name: str) -> Any:
+        """Forward missing attributes to the wrapped module."""
+        try:
+            return super().__getattr__(name)  # defer to nn.Module's logic
+        except AttributeError:
```

Comment:
Note for future: If when trying to get a property attribute `p`, `p`'s implementation raises an `AttributeError`, and `self.module` does not have `p` either, then the error message here will be non-intuitive since it would say that `self.module` does not have the attribute `p`, where the useful error would be that accessing `ddp_module.p` raised an `AttributeError` when accessing `.p`.